.github/workflows/pipeline.yml

```diff
@@ -0,0 +1,51 @@
+name: Test and Build
+
+on:
+  pull_request:
+    branches:
+  push:
+    branches:
+      - master
+
+jobs:
+  login:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Login into GitHub Container Registry
+        run: echo "${{ secrets.CR_PAT }}" | docker login https://ghcr.io -u ${{ github.actor }} --password-stdin
```

Comment:
So, the user making the PR logs in and tries to pull the container from your registry? I wonder if that will work if I tried a PR...

Reply:
I think if you push a change to this PR, your user will trigger the pipeline. Those images are public so it should still work if you have your personal access token. This is temporary until we can get the splunk images public